Sources/XCBBuildServiceProxy/Hybrid/HybridRPCRequestHandler.swift

```diff
@@ -40,7 +41,7 @@ final class HybridRPCRequestHandler<RequestHandler: HybridXCBBuildServiceRequest
         // Here we are receiving a request from Xcode
         let request = unwrapInboundIn(data)
         
-        logger.debug("Received RPCRequest from Xcode: \(request)")
+        os_log("Received RPCRequest from Xcode: \(request)")
```

Comment:
I'm not a fan of the `os_log` change. The difference between these is that `logger` outputs to stderr, while `os_log` is hidden away in the Console. Maybe XCBuild started to care about stderr for some sort of communication?

If we must use `os_log`, can we keep the levels (trace/debug = `os_log_debug`, info = `os_log_info`, etc.)?

Reply:
> I'm not a fan of the os_log change. The difference between these is that logger outputs to stderr, while os_log is hidden away in the Console. Maybe XCBuild started to care about stderr for some sort of communication?

Could be! I couldn't figure it out, just that it was blocking it from working.

> If we must use os_log, can we keep the levels (trace/debug = os_log_debug, info = os_log_info, etc.)?

I agree and I tried that, but those require MacOS 12, and we still have some folks on 11 (for now).

Reply:
` but those require MacOS 12` wait maybe this isn't right, gonna check what the issue was again

Reply:
Nevermind, I can update it with those

Reply:
@brentleyjones I updated the `os_log` calls with context. There wasn't an equivalent for `.trace` though

Reply:
`.trace` is even more verbose than `.debug`, so `os_log_debug`.

Reply:
roger that :+1: